### PR TITLE
Kickstart the Contractor program

### PR DESCRIPTION
**Pull request by jviotti**
See: https://github.com/orgs/json-schema-org/discussions/779

### CODE REVIEW

**Comment by jviotti** [comment]
@gregsdennis Looks like there is a CI issue when referencing a GitHub Action of yours. cc @benjagm 

Screenshot 2024-08-09 at 12 28 16 PM

**Comment by gregsdennis** [comment]
Yeah, I pushed an update that apparently broke things.  Working on it.

File: programs/contractors/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+The JSON Schema Contractor Program aims to promote official professional services around JSON Schema coming from TSC members and Ambassors, connecting companies with the key people they need to be successful with JSON Schema, while promoting the sustainability of JSON Schema organization.
+
+### How does the program work?
+
+- The TSC and existing ambassadors can opt-in to be listed as contractors at any time.
+- All contractor information will be displayed on the JSON Schema website.
+
+### Contractor requirements
+
+The participation will be determined and checked annually. The only requirement for being listed as a contractor is being a current TSC member or an current Ambassador.
```
Comment (gregsdennis) [inline programs/contractors/README.md:14]:
```suggestion
The participation will be determined and checked annually. The only requirement for being listed as a contractor is being a current TSC member or a current Ambassador.
```

File: programs/contractors/contractors.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "name": "Juan Cruz Viotti",
+    "github": "jviotti",
+    "linkedin": "jviotti",
+    "email": "jv@jviotti.com",
+    "website": "https://www.jviotti.com",
+    "type": "tsc",
+    "incorporatedIn": [ "UK", "US" ],
+    "bio": "I'm a TSC member of JSON Schema, did award-winning research at the University of Oxford in the space of binary serialization and compression using JSON Schema, co-authored an O'Reilly book covering JSON Schema, and own several projects in the ecosystem, ranging educational material like LearnJSONSchema.com to a high-performance C++ JSON Schema compiler and validator. I have experience helping organizations be successful with JSON Schema in the Data, API, and IoT spaces all the way from architecture and training to custom development. I'm eager to help you take your JSON Schema integrations to the next level!",
```
Comment (gregsdennis) [inline programs/contractors/contractors.json:10]: This is really good.  We should prop each other up by helping write these for each other.

**Review by gregsdennis — APPROVED** [review]
Just a minor grammatical correction.  Looks great!

File: programs/contractors/contractors.json (hunk 2 above)
Comment (jviotti) [reply programs/contractors/contractors.json:10]: Great to hear you liked it and happy to help with other bios if I can!

File: programs/contractors/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+This document covers the organizational and operational aspects of the JSON Schema Contractor Program.
+
+### What is the JSON Schema Contractor Program?
+
+The JSON Schema Contractor Program aims to promote official professional services around JSON Schema coming from TSC members and Ambassors, connecting companies with the key people they need to be successful with JSON Schema, while promoting the sustainability of JSON Schema organization.
```
Comment (jdesrosiers) [inline programs/contractors/README.md:5]:
```suggestion
The JSON Schema Contractor Program aims to promote official professional services around JSON Schema coming from TSC members and Ambassors, connecting companies with the key people they need to be successful with JSON Schema, while promoting the sustainability of the JSON Schema organization.
```

File: programs/contractors/contractors-schema.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      "website": {
+        "type": "string",
+        "format": "url",
+        "description": "Website URL of the contractor"
+      },
+      "github": {
+        "type": "string",
+        "description": "GitHub profile URL of the contractor"
+      },
+      "twitter": {
```
Comment (jdesrosiers) [inline programs/contractors/contractors-schema.json:31]: As much as I hate to say it, should this be "X" instead of "twitter"?

**Review by jdesrosiers — APPROVED** [review]
I just made a couple minor suggestions. I'll add my profile as soon as this is merged.

File: programs/contractors/contractors-schema.json (hunk 4 above)
Comment (gregsdennis) [reply programs/contractors/contractors-schema.json:31]: Actually make it the Unicode value `𝕏` 😆

Comment (gregsdennis) [reply programs/contractors/contractors-schema.json:31]: Why not support both?  It's that [Robustness Principle](https://en.m.wikipedia.org/wiki/Robustness_principle) thing.

Comment (jviotti) [reply programs/contractors/contractors-schema.json:31]: Ah, fair enough. I adapted this from the ambassador schema/readme, which still mentions Twitter

Comment (jviotti) [reply programs/contractors/contractors-schema.json]: Done!

**Pull request merged**